### PR TITLE
chore(actions): add workflow config

### PR DESCRIPTION
**Pull request by practicatto**
I added actions to this repo. I don't follow any conventions in the creation of branches because i think this is a project we do for fun so i like making puns but once we establish them i'll follow them. But we can keep it fun for now.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Code Quality
+
+on:
```
Comment (AlexVelezLl) [inline .github/workflows/ci.yml]: I personally prefer the `on: [push, pull_request]` syntax but its ok.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Code Quality
```
Comment (AlexVelezLl) [inline .github/workflows/ci.yml]: Is this workflow just for linting? I think we can rename the file also. and remove the last build step :thinking:

File: .github/workflows/ci.yml (hunk 1 above)
Comment (practicatto) [reply .github/workflows/ci.yml]: we can change it, it looks more neat in the way you're suggesting tho

File: .github/workflows/ci.yml (hunk 2 above)
Comment (AlexVelezLl) [reply .github/workflows/ci.yml]: nvm I guess is just a check to see if it builds xd, its okay then

Comment (practicatto) [reply .github/workflows/ci.yml]: por qué hablas en inglés? jaja si mi descripción ta en español mal trip but ok

Comment (AlexVelezLl) [reply .github/workflows/ci.yml]: lol

Comment (AlexVelezLl) [reply .github/workflows/ci.yml]: Fue automatico ajsdfasdfasdfjasdf, la costumbre

**Review by AlexVelezLl — APPROVED** [review]

**Comment by AlexVelezLl** [comment]
lgtm!

**Pull request merged**